dev-docs/bidders/premiumads.md

```diff
@@ -0,0 +1,21 @@
+---
+layout: bidder
+title: PremiumAds
+description: PremiumAds Bidder Adapter
+pbjs: true
+biddercode: premiumads
+media_types: banner
+gdpr_supported: true
+schain_supported: true
+usp_supported: true
+
```

Comment:
Can you please add the following meta fields as well

- `floors_supported`: `true` (as you support a floor parameter)
- `gvl_id` - as you state that gdpr is supported
- `safeframes_ok`: true/false - helps publishers to vet your adapter